vroom.env

```diff
@@ -0,0 +1,4 @@
+SENTRY_KAFKA_BROKERS_PROFILING="kafka:9092"
+SENTRY_KAFKA_BROKERS_OCCURRENCES="kafka:9092"
+SENTRY_BUCKET_PROFILES=file://localhost//var/lib/sentry-profiles
```

Comment:
awesome !
Looking forward to store profiles on something else than my drive.
it runs out of inodes too quickly.